storage/innobase/include/srv0srv.h

```diff
@@ -1112,6 +1105,17 @@ void undo_spaces_deinit();
 #ifdef UNIV_DEBUG
 struct SYS_VAR;
 
+/** Enables master key redo encryption.
+ * Doesn't depend on the srv_redo_log_encrypt variable. */
```

Comment:
And add that it is used by SET innodb_redo_log_encrypt = MK.

Reply:
added comments.